src/migrations/migration_steps.ts

```diff
@@ -420,6 +421,20 @@ migrationStepRegistry
       }
       return data;
     },
+  })
+  .add("migration_22", {
```

Comment:
If you want to do that you'd also want to create a migration for the revisions. 

TBH I wouldn't bother, the tables will lose their filters it's no big deal.

Reply:
Yup; either both the revision and data migration or none of them.
on one hand, the filtering is reset once you reload your sheet so you could drop them
on the other hand, if the main user only has read-only access to the sheet, it will piss them off...
I'd argue for adding a command migration :/

Why did the command had an optional config to begin with?

Reply:
There's no need for a command migration, `"hasFilters":true` is part of the command :wink:

Reply:
fuck, it's not part of the command when used from the menu :grimacing:

Reply:
> Why did the command had an optional config to begin with?

I believe my idea was to save space/network usage by having smaller revisions since most of the CREATE_TABLE would have the exact same DEFAULT_CONFIG in the payload. I also don't put it in the exported json. I'm saving the trees 😤